src/config.js

```diff
@@ -145,11 +145,11 @@ export default class Config {
 
   nonInteractive: boolean;
 
-  workspacesExperimental: boolean;
+  workspacesEnabled: boolean;
 
   //
   cwd: string;
-  worktreeFolder: ?string;
+  workspaceFolder: ?string;
```

Comment:
There's an ambiguity with this name, since it's not clear if it's the path to the workspace common root or the path to a specific workspace.